ios/RNSScreen.mm

```diff
@@ -251,6 +251,9 @@ - (void)setStatusBarHidden:(BOOL)statusBarHidden
   _statusBarHidden = statusBarHidden;
   [RNSScreenWindowTraits assertViewControllerBasedStatusBarAppearenceSet];
   [RNSScreenWindowTraits updateStatusBarAppearance];
+  // As the status bar could change its visibility, we need to calculate header height
+  // for the correct value in `onHeaderHeightChange` event.
+  [self.controller calculateAndNotifyHeaderHeightChangeIsModal:NO];
```

Comment:
Hey, I'm not convinced RN that hiding the status bar actually changes the height of a header. How does that work? Does it work like I think: hiding status bar has no effect on header height visually, but actual header height increases by height of the status bar?

Reply:
Youre right! 😄 this has been fixed in https://github.com/software-mansion/react-native-screens/pull/1917/commits/6e3cbb9b3a306a6bdb039282b8300ddf922efdbe and calling the method has been removed in https://github.com/software-mansion/react-native-screens/pull/1917/commits/46364e59f2d4b4f23186b7f519178d0c495a1955.